gcloud/logging/_gax.py

```diff
@@ -120,7 +120,12 @@ def logger_delete(self, project, logger_name):
         """
         options = None
         path = 'projects/%s/logs/%s' % (project, logger_name)
-        self._gax_api.delete_log(path, options)
+        try:
+            self._gax_api.delete_log(path, options)
+        except GaxError as exc:
+            if exc_to_code(exc.cause) == StatusCode.NOT_FOUND:
+                raise NotFound(path)
```

Comment:
In order to stick it into `NotFound._errors`, we would need to synthesize a mapping to hold that text.  Given that it adds no real information, I think we could skip that bit.